docs/content/integrations/airlift/reference.mdx

```diff
@@ -123,3 +123,55 @@ defs = Definitions.merge(
     ),
 )
 ```
+
+### Customizing DAG proxying operator
+
+Similar to how we can customize the operator we construct on a per-dag basis, we can customize the operator we construct on a per-dag basis. We can use the `build_from_dag_fn` argument of `proxying_to_dagster` to provide a custom operator in place of the default.
+
+For example, let's take a look at the following custom operator which expects an API key to be provided as a variable:
```

Comment:
When you phrase it as, "let's take a look", that makes me think there will be follow-up explanation.

Instead, can we explain what the user is to look for in the code sample below?

> For example, in the following example we can see that the operator is customized to provide an authorization header while authentication with Dagster